src/binding.cc

```diff
@@ -635,23 +618,25 @@ class Client : public ObjectWrap {
 
     void emit_done(my_ulonglong insert_id = 0, my_ulonglong affected_rows = 0,
                   my_ulonglong num_rows = 0) {
-      HandleScope scope;
-      Local<Object> info = Object::New();
-      info->Set(insert_id_symbol, Number::New(insert_id));
-      info->Set(affected_rows_symbol,
-                Number::New(affected_rows == (my_ulonglong) - 1
+      NanScope();
+
+      Local<Object> info = NanNew<Object>();
+      info->Set(NanNew<String>("insertId"), NanNew<Number>(insert_id));
+      info->Set(NanNew<String>("affectedRows"),
+                NanNew<Number>(affected_rows == (my_ulonglong) - 1
                              ? 0
                              : affected_rows));
```

Comment:
Style nit: The above 2 lines should be re-indented to match the changes preceding them.